### ISSUE TITLE
Using a custom SSL Certificate doesn't seem to work

### ISSUE THREAD

**Issue by jrisebor** [omega8cc/boa#989]
I tried setting up a custom SSL Certificate using the instructions found in SSL.txt, but it keeps serving an old let's encrypt certificate that failed to create properly. 

I have created the empty `dont-overwrite-example.com.pid` file, but when I run a verify a new `demo-www.example.com.pid` gets created with the contents `www.example.com`

What do I need to do to get BOA to serve the custom certificate?

**Comment by omega8cc** [comment]
>> old let's encrypt certificate that failed to create properly

You need to first make it work with LE cert and only then attempt to replace it.

**Comment by omega8cc** [comment]
Also, you must enable live mode.

**Comment by omega8cc** [comment]
Plus, if your site name is `www.example.com` then the control file should have a filename `dont-overwrite-www.example.com.pid`

**Comment by jrisebor** [comment]
> You need to first make it work with LE cert and only then attempt to replace it.
The LE cert won't work because I'm testing this with a hosts file to make sure it works before moving to it, so the DNS records aren't what LE is expecting. Why does an LE cert need to be made first?

>Also, you must enable live mode.
I tried this, didn't seem to make a difference, should probably be noted in the docs more clearly

>Plus, if your site name is www.example.com then the control file should have a filename dont-overwrite-www.example.com.pid
My bad, I made a typo in my post. The file is correctly named following that convention.
